tests/test_audio_48000kHz.py

```diff
@@ -0,0 +1,58 @@
+import unittest
+from unittest.mock import patch, MagicMock
+from facefusion.ffmpeg import restore_audio
+
+class TestRestoreAudio(unittest.TestCase):
+
+    @patch('facefusion.ffmpeg.state_manager')
+    @patch('facefusion.ffmpeg.get_temp_file_path')
+    @patch('facefusion.ffmpeg.run_ffmpeg')
+    def test_restore_audio(self, mock_run_ffmpeg, mock_get_temp_file_path, mock_state_manager):
```

Comment:
I would actual say, just write a cli based integration test - this looks over the top with all the mocks